samcli/commands/local/cli_common/options.py

```diff
@@ -117,6 +145,20 @@ def invoke_common_options(f):
                 help="Specifies the location basedir where the Layers your template uses will be downloaded to.",
                 default=get_default_layer_cache_dir(),
             ),
+            click.option(
+                "--rapid-basedir",
```

Comment:
Do we want to expose this functionality?

Reply:
Will look at an alternative approach tomorrow, but short answer is "yes" - if the user has folder permission issues and we don't provide this option, they're basically stuck with no ability to invoke. However, we should carefully document that most users will never want to touch this.